crates/core/src/generated/mod.rs

```diff
@@ -165,12 +261,12 @@ impl PeerChange<'_> {
 
     pub fn added_connection_msg(
         transaction: Option<impl AsRef<str>>,
-        (from, from_location): (PeerId, f64),
-        (to, to_location): (PeerId, f64),
+        (from, from_location): (String, f64),
```

Comment:
wouldn't it be better to still pass the typed PeerId here and convert to string inside the function for clarity? is this called from core? if so then it should be possible

Reply:
I had to change it because it's used on fdev too. And from there you have to build it from the fbs string. After the removal of `PeerId::from_str` we cannot build any PeerId object from `fdev/src/network_metrics_server.rs` that's where is also used.

Reply:
makes sense